packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -1,48 +1,57 @@
 <ui5-dialog
+	dir="{{effectiveDir}}"
```

Comment:
You should import ```effectiveDir``` before using it.

Reply:
Maybe not necessary because it has built-in functionality?